src/main/java/com/conveyal/taui/controllers/RegionalAnalysisController.java

```diff
@@ -79,34 +83,107 @@ public static RegionalAnalysis deleteRegionalAnalysis (Request req, Response res
     }
 
     /**
-     * Get a particular percentile of a query as a grid file.
-     * FIXME this does not seem to do that anymore. It just gets the single percentile that exists for any one analysis.
+     * The client usually fetches gzipped grids of regional analysis results from S3 when the analysis is done.
+     * This endpoint allows fetching the partially completed grid from the backend while the analysis is still in
+     * progress.
      */
-    public static Object getPercentile (Request req, Response res) throws IOException {
+    public static InputStream getPartialRegionalAnalysisResult (Request req, Response res) {
```

Comment:
Where is this method used?

Reply:
Same question 🤔

Reply:
This is dead code and should be removed. It was the original implementation with two different endpoints. I merged this code into `getRegionalResults()`.